commands/background.js

```diff
@@ -0,0 +1,9 @@
+browser.commands.getAll(function(commands) {
```

Comment:
This won't work in Chrome because it uses "browser", not "chrome". I think, if we can, we should be cross-browser here.